cli/flags.rs

```diff
@@ -98,7 +98,7 @@ pub enum DenoSubcommand {
   Test {
     doc: bool,
     no_run: bool,
-    fail_fast: bool,
+    fail_fast: usize,
```

Comment:
Make this an option rather than relying on the value being falsy.

```suggestion
    fail_fast: Option<usize>,
```

Reply:
Ok fixed